tmt/convert.py

```diff
@@ -652,7 +682,9 @@ def read_nitrate(beaker_task, common_data, disabled, general):
     return common_data, individual_data
 
 
-def extract_relevancy(notes, field):
+def extract_relevancy(
+        notes: str,
+        field: tmt.utils.StructuredField) -> Optional[Union[str, List[str]]]:
```

Comment:
@ukulekek could we add a type alias for `Union[str, List[str]]`, maybe call it `RelevancyType` ?